cli/index.js

```diff
@@ -306,28 +306,40 @@ function run(paths) {
   ).then(async values => {
     console.log(chalk.green(`Built ${values.length} documents.`));
     const titles = {};
```

Comment:
Ignore my comment about `titles[locale].titles = JSON.parse(...)` around line 319. All you need to do is move this inside the loop over `titlesByLocale`.